src/Services/PageService/Entities/SuccessPageEntity.cs

```diff
@@ -30,6 +30,6 @@ public class SuccessPageEntity
         public bool DisplayBreadcrumbs { get; set; }
 
         public List<Breadcrumb> Breadcrumbs { get; set; }
-
+        public bool IsInPreviewMode { get; set; }
```

Comment:
Should there be an empty line above this (or remove the empty lines between the props above)